lib/openstudio-standards/standards/necb/NECB2011/system_fuels.rb

```diff
@@ -12,9 +12,12 @@ class SystemFuels
   attr_accessor :necb_reference_hp
   attr_accessor :necb_reference_hp_supp_fuel
   attr_accessor :fan_type
-  attr_accessor :swh_fueltype
   attr_accessor :ecm_fueltype
-  def set_defaults(standards_data:, primary_heating_fuel:)
+  attr_accessor :shw_fuel 
+  attr_accessor :shw_fueltype
+  attr_accessor :swh_fueltype
```

Comment:
why both s**hw**_fueltype and s**wh**_fueltype?

Reply:
Revised. Now we only have swh_fueltype as suggested.